packages/components/addon/components/hds/form/masked-input/field.hbs

```diff
@@ -17,7 +17,7 @@
     <Hds::Form::MaskedInput::Base
```

Comment:
I _think_ we should also pass down the `@visibilityToggleAriaLabel` and `@visibilityToggleAriaMessageText`, right?

Reply:
indeed, pushed a commit to forward them and added tests to cover these